packages/arb-ts/src/lib/bridge.ts

```diff
@@ -32,6 +35,14 @@ export class Bridge extends L2Bridge {
     ethSigner: Signer,
     arbSigner: Signer
   ) {
+    Promise.all([ethSigner.getAddress(), arbSigner.getAddress()]).then(
+      ([ethSignerAddress, arbSignerAddress]) => {
+        if (ethSignerAddress !== arbSignerAddress) {
+          throw new Error('L1 & L2 wallets must be of the same address')
```

Comment:
Does the library not work if we break this assumption?
I think everything still works with different signers.